examples/direct_fidelity_estimation.py

```diff
@@ -109,6 +174,8 @@ def direct_fidelity_estimation(circuit: cirq.Circuit, qubits: List[cirq.Qid],
         qubits: The list of qubits.
         noise: The noise model when doing a simulation.
         n_trial: The total number of Pauli measurements.
+        n_clifford_trials: In case the circuit is Clifford, we specify the
+            number of trials to estimate the noise-free pauli traces.
```

Comment:
offer a little more guidance on how to pick this? 

Also open up a follow-up issue for the following enhancement: DFE shows that you can do monte-carlo (random) sampling of the pauli stabilizers for a clifford circuit allowing you to estimate the fidelity of an n-qubit clifford state in O(1) time instead of doing it "exhaustively" which is O(2^n) [IIRC]. However, these asymptotic arguments really only come into play for "large" n because you still have to monte-carlo sample a lot of terms [for some definition of "a lot"]. If you need to do 500 trials, then log2(500) ~ 9 so for n < 9, you might as well do the exhaustive approach. So as a follow up, there are a couple of avenues to pursue

 - add an option for "exhaustive" sampling of the terms, which can be faster at low `n`.
 - do a numerical study to figure out what a reasonable value for `n_clifford_trials` is (which gives good convergence but isn't too large, which would mean a long runtime on the quantum processor)
 - [after those two] tell the user when they would be better off switching from exhaustive to monte-carlo

Reply:
Opened issue #2802 and added a TODO (but not right at this line of the code, but just above the definition of the flag).